colrev/ops/built_in/search_sources/aisel.py

```diff
@@ -380,17 +382,66 @@ def get_masterdata(
     def load(self, load_operation: colrev.ops.load.Load) -> dict:
         """Load the records from the SearchSource file"""
 
+        enl_mapping = {
+            ENTRYTYPES.ARTICLE: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+            ENTRYTYPES.INPROCEEDINGS: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+        }
+
+        entrytype_map = {
+            "Journal Article": ENTRYTYPES.ARTICLE,
+            "HICSS": ENTRYTYPES.INPROCEEDINGS,
+            "ICIS": ENTRYTYPES.INPROCEEDINGS,
+            "ECIS": ENTRYTYPES.INPROCEEDINGS,
+            "AMCIS": ENTRYTYPES.INPROCEEDINGS,
+            "Proceedings": ENTRYTYPES.INPROCEEDINGS,
+            "Inproceedings": ENTRYTYPES.INPROCEEDINGS,
+            "All Sprouts Content": ENTRYTYPES.INPROCEEDINGS,
+        }
+
         # pylint: disable=colrev-missed-constant-usage
         if self.search_source.filename.suffix in [".txt", ".enl"]:
             enl_loader = colrev.ops.load_utils_enl.ENLLoader(
                 load_operation=load_operation,
                 source=self.search_source,
+                list_fields={"A": " and "},
                 unique_id_field="ID",
             )
-            entries = enl_loader.load_enl_entries()
-            for entry in entries.values():
-                entry["ID"] = entry[Fields.URL].replace("https://aisel.aisnet.org/", "")
-            records = enl_loader.convert_to_records(entries=entries)
+            records = enl_loader.load_enl_entries()
+
+            for record_dict in records.values():
+                self.__fix_entrytype_before_conversion(record_dict=record_dict)
+                enl_loader.apply_entrytype_mapping(
+                    record_dict=record_dict, entrytype_map=entrytype_map
+                )
+                enl_loader.map_keys(record_dict=record_dict, key_map=enl_mapping)
+                record_dict["ID"] = record_dict[Fields.URL].replace(
+                    "https://aisel.aisnet.org/", ""
+                )
+            # records = enl_loader.convert_to_records(entries=records, enl_mapping=enl_mapping)
```

Comment:
please remove the code in comments and remove the %0 (type) field.

Reply:
Removed, I was thinking, as most of this is common across multiple loader, I can refactor all of them to a single function and use that. But lost track of it.